Makefile

```diff
@@ -11,13 +16,32 @@ GO_LDFLAGS=-ldflags "-X main.version=$(VERSION) \
 			-X main.buildHash=$(GITCOMMIT) \
 			-X main.buildDate=$(BUILDDATE)"
 
-.PHONY: all test
+.PHONY: all test clean-mocks mocks
 
 all: test
 
 test: 				## Run code tests
 	go test -v ./...
 
+test-cover: | mocks				## Run code tests with resources coverage
+	go test -coverpkg=./pkg/resource/... -covermode=count -coverprofile=coverage.out ./...
+	go tool cover -func=coverage.out
+
+clean-mocks:	## Remove mocks directory
+	rm -r mocks
```

Comment:
you have a target to clean it, should this directory be added to .gitignore instead?

Reply:
The elasticache team has this directory added to their git repository, and I included it as well for the sagemaker-controller repository. 

It is also used as an import for manager_test_suite_test.go:
mocksvcsdkapi “github.com/aws-controllers-k8s/sagemaker-controller/mocks/aws-sdk-go/sagemaker”
So it seems we need it in our git repository.